larray_editor/editor.py

```diff
@@ -873,38 +873,34 @@ def ipython_cell_executed(self):
                 self.select_array_item(last_input)
             return
 
-        # check if expression of the kind '<varname>[(...)] (...)' or '<varname>.<attribute> (...)'
-        varname = itemname = None
-        m = getitem_pattern.match(last_input)
```

Comment:
I **don't know** whether it is actually needed to check for setXXX patterns explicitly (previously the idea was to have setXXX for free from the getXXX pattern IIRC), but not checking at all for the getXXX pattern will introduce a slight regression AFAICT : it will not select the array (in the tree widget) if you type "income['M']" but will select it if you type "income". Did you intentionally remove that feature or was it an oversight?

Reply:
Hehe, I just checked and this didn't work previously. :wink: I wonder if we should support that or not.

Reply:
```python
arr1 = ndtest((3, 4))
arr2 = ndtest((3, 4, 5))
arr1['b0']
```